packages/generator/src/subtask2/my-func.test.ts

```diff
@@ -0,0 +1,33 @@
+import {createNewColumn} from './my-func';
+import ts from 'typescript';
+test('my own test', () => {
```

Comment:
Please, provide here some assertion or at least console.log to double check the output.